### ISSUE TITLE
[BUG] Abacus shift-scrolling doesn't work when sneak is bound to left control

### ISSUE THREAD

**Issue by object-Object** [FallingColors/HexMod#179]
Basically what the title says. Because "shift" scrolling uses the sneak keybind but "ctrl" scrolling is hardcoded to ctrl instead of using the run keybind, you can't set the abacus in increments of 1 or 0.1 if you set your sneak key to ctrl.

**Comment by yrsegal** [comment]
I don't think this is a keybind problem - the "holding ctrl" thing is just checking if the modifier key is being pressed. Which means command on mac, and ctrl on windows. So you can't rebind that, not really...

**Comment by walksanatora** [comment]
Which is why it should probably be the crouch and sneak keys
Since some people (like me) bind spring to shift and crouch to control

**Comment by yrsegal** [comment]
that's up to petra to decide then, not me

**Comment by gamma-delta** [comment]
What do you mean by the "crouch and sneak keys?" Those are the same key.

... The irritating thing is that there's not any easy fix to this, because if I add a remappable keybind for the "ctrl" behavior, then it will conflict with anyone else trying to use that key, including vanilla's sprint keybind.

I suppose *a* solution is to have some config option to remap it to a hard-coded different key, like `alt`. To be honest, that might be the best solution even though it sucks. But I'd like your feedback on what to do.

**Comment by walksanatora** [comment]
sorry i meant that people (like me) like to swap the crouch and **sprint** (wrong s word)
so i think a better option would be to detect if holding the sprint key instead of checking ctrl specifically (which is the default value for the sprint key)

**Comment by ChloeDawn** [comment]
I am also encountering this issue from having sneak bound to Ctrl and sprint bound to Shift.
It would be nice if the mod added a keybinding for the modifier to give players more configurability.

**Comment by Lykrast** [comment]
As another shift for spring ctrl for sneak, I agree that's an issue.

If making an actual keybind is an issue, then yeah a line in the client config to switch it would be good enough imo. Could probably hardcode it to 3 values so users can choose between ctrl, shift, and alt, something like that.

**Comment by lv00-player** [comment]
This is a problem for me as well.
Please make make the default behaviour use the sprint key instead of the control modifier.
It wouldn't change anything for shift-sneak users as ctrl is sprint by default, and would allow easy customisation for anyone else.

**Comment by object-Object** [comment]
If this isn't implemented yet, we should fix it for the 1.20.1 release.
